### PR TITLE
Enabled Github Action for CLA check for Oppia-Android

### PR DESCRIPTION
**Pull request by gp201**
## Explanation

I have done the following in this PR:

1. Enabled Github Action for CLA Check for Oppia Android
2. Removed the requirement to have GITHUB_TOKEN as an env variable in the workflow.
3. Added messages in the `dispatcher.js` to make it clear which check is being executed.

**Note:** I have tested the changes in another repository already.

**Note:** The 200,000+ changes in `actions_build/index.js` are expected. These changes are autogenerated when the changes were made to the `actions/src/pull_requests/claCheck.js` file. The [initial commit](https://github.com/oppia/oppiabot/pull/267/commits/e91aeb0b44b2db2dc9c0b8cec3dba01929569836) is where these changes occured

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: actions/src/dispatcher.js [hunk 1]
```diff
@@ -35,9 +35,11 @@ module.exports = {
         for (var i = 0; i < checkList.length; i++) {
           switch (checkList[i]) {
             case constants.issuesLabelCheck:
+              core.info('issue label check triggered');
```
Comment (vojtechjelinek) [inline actions/src/dispatcher.js:38]: Do we need this?

File: actions/src/dispatcher.js [hunk 2]
```diff
@@ -35,9 +35,11 @@ module.exports = {
...
               await issueLabelsModule.checkLabels();
               break;
             case constants.claCheckGithubAction:
+              core.info('cla check triggered');
```
Comment (vojtechjelinek) [inline actions/src/dispatcher.js:42]: Do we need this?

File: actions/src/pull_requests/claCheck.js [hunk 3]
```diff
@@ -43,14 +43,24 @@ const authorize = async function() {
...
   const octokit = new GitHub(GITHUB_TOKEN);
   const PR_NUMBER = context.payload.pull_request.number;
   const PR_AUTHOR = context.payload.pull_request.user.login;
+  const REPO_NAME = context.payload.repository.name;
+
+  let LINK_RESULT = '';
+
+  if (REPO_NAME === 'oppia-android' ){
+    LINK_RESULT = 'here'.link(
+      'https://github.com/oppia/oppia-android/wiki#onboarding-instructions'
```
Comment (vojtechjelinek) [inline actions/src/pull_requests/claCheck.js:56]: Hmm, I have seen this in other parts of the codebase, could you check if we already have these links defined somewhere?

File: constants.js [hunk 4]
```diff
@@ -62,6 +62,8 @@ const checksWhitelist = {
   'oppia-android': {
     [openEvent]: [claCheck],
     [reopenEvent]: [],
+    [openEventGithubActions]: [claCheckGithubAction],
+    [reopenEventGithubActions]: [claCheckGithubAction],
```
Comment (vojtechjelinek) [inline constants.js:66]: Why are these added?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: actions/src/dispatcher.js (hunk 1 above)
Comment (gp201) [reply actions/src/dispatcher.js:38]: This is to make it clear what is being executed in the GitHub Action. Since we have one workflow for both the checks (Issue label check and CLA Check). Will also make debugging easier in the future.

File: actions/src/dispatcher.js (hunk 2 above)
Comment (gp201) [reply actions/src/dispatcher.js:42]: This is to make it clear what is being executed in the GitHub Action. Since we have one workflow for both the checks (Issue label check and CLA Check). Will also make debugging easier in the future.

File: constants.js (hunk 4 above)
Comment (gp201) [reply constants.js:66]: This is meant to enable The CLA Check GitHub action for Oppia-Android

File: actions/src/pull_requests/claCheck.js (hunk 3 above)
Comment (gp201) [reply actions/src/pull_requests/claCheck.js:56]: It's not there in the codebase. It is present in https://github.com/oppia/oppiabot/pull/257#discussion_r620301095 this comment.

**Comment by gp201** [comment]
@vojtechjelinek PTAL

**Review by jameesjohn — APPROVED** [review]

**Review by vojtechjelinek — APPROVED** [review]
LGTM!
